manifests/init.pp

```diff
@@ -17,7 +17,11 @@
 class cobald(
   String                     $cobald_version             = undef,                        # cobald version to be used (undef = latest PyPI release, 'master' = Github master branch or PyPI release number)
   String                     $tardis_version             = undef,                        # tardis version to be used (undef = latest PyPI release, 'master' = Github master branch or PyPI release number)
```

Comment:
Setting `cobald_version` or `tardis_version` to `master` or any other branch name will not work, if an installation from PyPi is requested. These cases need to be caught.

Another question is what happens if one leaves `cobald_version` or `tardis_version` to `undef` and chooses to install from git. Probably the best option would be to use the `master` branch. @stefan-k, @olifre: What is your opinion on this?

The comment needs to be updated to reflect the new situation.

Reply:
> Another question is what happens if one leaves `cobald_version` or `tardis_version` to `undef` and chooses to install from git.

Since the current implementation does not specify `Optional`, it can never be set to `undef`. On the other hand, I agree that changing that might be a good idea: `master` or the latest version from `PyPI` seem like sensible defaults.